modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/web/socket/WebSocketsManager.java

```diff
@@ -476,5 +481,19 @@ boolean canBeClose() {
         public void setClusterIds(Set<String> clusterIds) {
             this.clusterIds = clusterIds;
         }
+
+        /**
+         * @param hasDemo Has demo flag.
+         */
+        public void setHasDemo(Boolean hasDemo) {
```

Comment:
Lets use primitive 'boolean', not 'Boolean'.

Reply:
Changed.